src/jdk.incubator.concurrent/share/classes/jdk/incubator/concurrent/StructuredTaskScope.java

```diff
@@ -269,17 +269,21 @@
     private final ThreadFlock flock;
     private final ReentrantLock shutdownLock = new ReentrantLock();
 
-    // the set of "tracked" Future objects, created lazily
+    // the set of "tracked" Future objects waiting to be returned by Future.get, created lazily
+    // assigned once-only with FUTURES.compareAndSet, read by any thread
```

Comment:
Instead of "assigned once-only ..." it might be simpler to just link to the track method

Reply:
Changed to "    // assigned to non-null value in method track, read by any thread"